password_security/models/res_users.py

```diff
@@ -41,6 +44,31 @@ def write(self, vals):
         return super(ResUsers, self).write(vals)
 
     @api.multi
+    def password_match_message(self):
+        self.ensure_one()
+        company_id = self.company_id
+        message = []
+        if company_id.password_lower:
+            message.append('\n* ' + _('Lowercase letter (At least ' + str(
+                company_id.password_lower) + ' character)'))
+        if company_id.password_upper:
+            message.append('\n* ' + _('Uppercase letter (At least ' + str(
+                company_id.password_upper) + ' character)'))
+        if company_id.password_numeric:
+            message.append('\n* ' + _('Numeric digit (At least ' + str(
+                company_id.password_numeric) + ' character)'))
+        if company_id.password_special:
+            message.append('\n* ' + _('Special character (At least ' + str(
+                company_id.password_special) + ' character)'))
```

Comment:
You can't use `_()` like this. Check the warning in the docs: https://www.odoo.com/documentation/11.0/reference/translations.html#explicit-exports